pipelines/scheduled/platforms/build_windows.no_gpl.arches

```diff
@@ -1,5 +1,5 @@
-# OS       TRIPLET                    ARCH       DOCKER_ARCH    MAKE_FLAGS    TIMEOUT     DOCKER_TAG
-windows    x86_64-w64-mingw32nogpl    x86_64     x86_64         VERBOSE=1     .           v5.44
+# OS       TRIPLET                    ARCH       DOCKER_ARCH    MAKE_FLAGS                                              TIMEOUT     DOCKER_TAG
+windows    x86_64-w64-mingw32nogpl    x86_64     x86_64         USE_GPL_LIBS=0,JL_CFLAGS=-Werror,JL_CXXFLAGS=-Werror    .           v5.44
```

Comment:
I tried to fix some of the warnings, but it's more complicated than I thought.
Maybe we won't be able to fix all the warnings in a short time.

How about adding a TODO and moving on?


```suggestion
# OS       TRIPLET                    ARCH       DOCKER_ARCH    MAKE_FLAGS      TIMEOUT     DOCKER_TAG
windows    x86_64-w64-mingw32nogpl    x86_64     x86_64         USE_GPL_LIBS=0  .           v5.44
# TODO: fix gcc warning, and add flags `JL_CFLAGS=-Werror,JL_CXXFLAGS=-Werror` to MAKE_FLAGS
```